src/styles/landing-page.less

```diff
@@ -44,14 +44,19 @@
   background-color: @landing-side-bar-bg;
   color: @color-pf-white;
   @media(min-width: @screen-sm-min) {
-    flex: 0 1 auto;
-    width: 325px;
+    bottom: 0;
+    overflow: hidden;
+    overflow-y: auto;
+    position: fixed;
+    right: 0;
```

Comment:
@jeff-phillips-18 I agree there's a problem here. Do you mind if we make it a follow-on? The right fix isn't obvious to me. It feels like a problem because two scrollbars are competing for the same place on the screen, and potentially we need to test many different browsers and platforms.